src/comm.c

```diff
@@ -225,8 +219,8 @@ send_plug_info (int soc, const char *oid)
     }
 
   send_printf
-   (soc, "%s <|> %s <|> %d <|> %s <|> %s <|> %s <|> %s <|> %s <|> %s\n",
```

Comment:
Hey Hani, how about removing the field entirely, so we don't have to parse the dummy field on Manager side?

In other words, remove the "<|>" too.

People aren't supposed to be using OTP directly, so only Manager would need to change, right?

Reply:
Hey matt, I agree. Updated the PR to remove it.